src/webhint-theme/source/js/details-polyfill.js

```diff
@@ -1,80 +1,84 @@
-/**
- * Original code by Roc Sta. Cruz.
- * Distributed under the MIT License
- * https://github.com/rstacruz/details-polyfill/releases/tag/v1.1.0
- */
-/*eslint-disable*/
-
-void (function (root, factory) {
-    if (typeof define === 'function' && define.amd) define(factory)
-    else if (typeof exports === 'object') module.exports = factory()
-    else factory()
-  }(this, function () {
-    var DETAILS = 'details'
-    var SUMMARY = 'summary'
-
-    var supported = checkSupport()
-    if (supported) return
-
-    // Add a classname
-    document.documentElement.className += ' no-details'
-
-    window.addEventListener('click', clickHandler)
-
-    injectStyle('details-polyfill-style',
-      'html.no-details ' + DETAILS + ':not([open]) > :not(' + SUMMARY + ') { display: none; }\n' +
-      'html.no-details ' + DETAILS + ' > ' + SUMMARY + ':before { content: "\u25b6"; display: inline-block; font-size: .8em; width: 1.5em; }\n' +
-      'html.no-details ' + DETAILS + '[open] > ' + SUMMARY + ':before { content: "\u25bc"; }')
-
-    /*
-     * Click handler for `<summary>` tags
-     */
-
-    function clickHandler (e) {
-      if (e.target.nodeName.toLowerCase() === 'summary') {
-        var details = e.target.parentNode
-        if (!details) return
-
-        if (details.getAttribute('open')) {
-          details.open = false
-          details.removeAttribute('open')
-        } else {
-          details.open = true
-          details.setAttribute('open', 'open')
+/* global setImmediate */
```

Comment:
Same comments as in https://github.com/webhintio/hint/pull/1326 for this file (although here you already have a better name 😊)